target_chains/ethereum/contracts/forge-test/Entropy.t.sol

```diff
@@ -115,25 +118,15 @@ contract EntropyTest is Test, EntropyTestUtils, EntropyEvents {
         uint fee,
         address provider,
         uint randomNumber,
-        bool useBlockhash
+        bool useBlockhash,
+        bytes4 revertReason
     ) public {
-        // Note: for some reason vm.expectRevert() won't catch errors from the request function (?!),
-        // even though they definitely revert. Use a try/catch instead for the moment, though the try/catch
-        // doesn't let you simulate the msg.sender. However, it's fine if the msg.sender is the test contract.
-        bool requestSucceeds = false;
-        try
-            random.request{value: fee}(
-                provider,
-                random.constructUserCommitment(bytes32(uint256(randomNumber))),
-                useBlockhash
-            )
-        {
-            requestSucceeds = true;
-        } catch {
-            requestSucceeds = false;
-        }
-
-        assert(!requestSucceeds);
+        bytes32 userCommitment = random.constructUserCommitment(
+            bytes32(uint256(randomNumber))
+        );
+        vm.deal(address(this), fee);
```

Comment:
doh... is this why the revert thing wasn't working before?

Reply:
yes, it was expecting a revert inside `random.constructUserCommitment`